DataFormats/SiStripCluster/src/classes_def.xml

```diff
@@ -1,12 +1,19 @@
 <lcgdict>
 
- <class name="SiStripCluster" ClassVersion="13">
+ <class name="SiStripCluster" ClassVersion="14">
+  <version ClassVersion="14" checksum="1374720584"/>
   <version ClassVersion="13" checksum="1374720584"/>
   <version ClassVersion="12" checksum="2984011925"/>
   <version ClassVersion="11" checksum="3702468681"/>
   <version ClassVersion="10" checksum="3791198690"/>
   <field name="error_x" transient="true"/>
  </class>
+ <ioread sourceClass="SiStripCluster" version="[10]" targetClass="SiStripCluster" source="std::vector<uint8_t> amplitudes_; uint16_t firstStrip_;" target="amplitudes_,firstStrip_,barycenter_,charge_" >
+   <![CDATA[amplitudes_ = onfile.amplitudes_; firstStrip_ = onfile.firstStrip_; newObj->initQB();]]>
+ </ioread>
+ <ioread sourceClass="SiStripCluster" version="[11-13]" targetClass="SiStripCluster" source="std::vector<uint8_t> amplitudes_; uint16_t firstStrip_; float barycenter_; int charge_;" target="amplitudes_,firstStrip_,barycenter_,charge_" >
+   <![CDATA[amplitudes_ = onfile.amplitudes_; firstStrip_ = onfile.firstStrip_; barycenter_ = onfile.barycenter_; charge_ = onfile.charge_; if (onfile.barycenter_ == 0) { newObj->initQB(); } else { firstStrip_ |= SiStripCluster::approximateMask; }]]>
+ </ioread>
```

Comment:
@pcanal Could you check and comment if there would any way to simplify the read rules above?

Reply:
This looks already minimal.  The only minor change would be to use:
```
if (onfile.barycenter_ == 0) { newObj->initQB(); } else { barycenter_ = onfile.barycenter_; charge_ = onfile.charge_; firstStrip_ |= SiStripCluster::approximateMask; }
```
since `initQB` already sets `charge_` and `firstStrip_`.